target/linux/mediatek/dts/mt7981a-ubnt-unifi-6-plus.dts

```diff
@@ -35,13 +37,19 @@
 	leds {
 		compatible = "gpio-leds";
 
-		led_blue: dome-blue {
+		led_blue: led-0 {
 			label = "blue:dome";
+			color = <LED_COLOR_ID_BLUE>;
+			function = LED_FUNCTION_STATUS;
+			function-enumerator = <0>;
```

Comment:
Why have `function-enumerator` when the color is different (white vs. blue)?

Reply:
Agree, this is unnecessary.

Reply:
It looks too puzzling to fix all this without knowing the features of specific devices. I think I'll limit myself to the device(s) I have. Additionally, not all features and colors are present in the kernel. For example, orange color or mesh function.

Reply:
LTE function is missing too. But for orange you have LED_COLOR_AMBER.